.test-infra/jenkins/common_job_properties.groovy

```diff
@@ -273,4 +259,91 @@ class common_job_properties {
         shell("python PerfKitBenchmarker/pkb.py $pkbArgs")
     }
   }
+
+  static def setPipelineJobProperties(def context, int tout, String descriptor) {
+    context.parameters {
+      stringParam(
+              'ghprbGhRepository',
+              'N/A',
+              'Repository name for use by ghprb plugin.')
+      stringParam(
+              'ghprbActualCommit',
+              'N/A',
+              'Commit ID for use by ghprb plugin.')
+      stringParam(
+              'ghprbPullId',
+              'N/A',
+              'PR # for use by ghprb plugin.')
+
+    }
+
+    // Set JDK version.
+    jdk('JDK 1.8 (latest)')
+
+    // Restrict this project to run only on Jenkins executors as specified
+    label('beam')
+
+    // Execute concurrent builds if necessary.
+    concurrentBuild()
+
+    wrappers {
+      timeout {
+        absolute(tout)
+        abortBuild()
+      }
+      credentialsBinding {
+        string("COVERALLS_REPO_TOKEN", "beam-coveralls-token")
+      }
+      downstreamCommitStatus {
+        context("Jenkins: ${descriptor}")
+        triggeredStatus("${descriptor} Pending")
+        startedStatus("Running ${descriptor}")
+        statusUrl()
+        completedStatus('SUCCESS', "${descriptor} Passed")
+        completedStatus('FAILURE', "${descriptor} Failed")
+        completedStatus('ERROR', "Error Executing ${descriptor}")
+      }
+      // Set SPARK_LOCAL_IP for spark tests.
+      environmentVariables {
+        env('SPARK_LOCAL_IP', '127.0.0.1')
+      }
+    }
+
+    // Set Maven parameters.
+    setMavenConfig(context)
+
+  }
+
+  static def setPipelineBuildJobProperties(def context) {
```

Comment:
Not obvious from the names what is different between `setPipelineJobProperties` and `setPipelineBuildJobProperties`.